extensions/2.0/Vendor/EXT_materials_clearcoat_color/README.md

```diff
@@ -0,0 +1,131 @@
+
+```json
+{
+    "materials": [
+        {
+            "name": "varnish",
+            "extensions": {
+                "KHR_materials_clearcoat": {
+                    "clearcoatFactor": 1.0,
+                    "extensions": {
+                        "EXT_materials_clearcoat_color": {
+                            "clearcoatColorFactor": [
+                                0.4,
+                                0.1,
+                                0.1
+                            ]
+                        }
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+### Clearcoat Color Parameters
+
+All implementations should use the same calculations for the input parameters. Implementations of the BRDF itself can vary based on device performance and resource constraints. See [Appendix B](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#appendix-b-brdf-implementation) for more details on the BRDF calculations.
+
+As with the core glTF 2.0 spec, all coloured textures are assumed to be sRGB and all coloured factors are assumed to be linear.
+
+|                                  | Type                                                                            | Description                            | Required             |
+|----------------------------------|---------------------------------------------------------------------------------|----------------------------------------|----------------------|
+|**clearcoatColorFactor**    | `number[3]`                                                                                       | The clearcoat's colour factor.  | No, default: `[1.0, 1.0, 1.0]`   |
+|**clearcoatColorTexture**   | [`textureInfo`](https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo)  | The clearcoat layer's color texture.  | No                               |
+
+As with other factor/texture combinations, these two values are multiplied before being used in calculations. If not defined, the defaults represent a white transmission color and, therefore, no absorption.
+```
+clearcoat_transmission_color = clearcoatColorFactor * clearcoatColorTexture
+```
+
+## Physical Interpretation
+
+The clearcoat is a thin, dielectric layer that specularly-transmits light through it but, by default, doesn't absorb light. This extension allows us to specify the observed color tint of light passing through the coating from the underlying layer. The `clearcoat_transmission_color` represents the **square of transmittance of the coating**. This is because the observed color tint of the underlying material results after light has passed through the coating twice (in incoming and outgoing directions), making the final color more easily artist-directable.
```

Comment:
Just for understanding, does the squared value also have impact on texture precision if 8-bit? Similar to gamma space?

Reply:
No more than any other coloured texture. The texture itself will be stored as sRGB, just like any coloured texture and then linearized before using the value in calculations. The colour of the texture represents the final, observed look of the tinting. For the sake of path tracers, we specify that what is actually happening in the coat is the light ray passing through the coat twice, once on the way in and once on the way out. Therefore, the tint colour represents the square of the transmittance.